test-framework/core/src/main/java/org/keycloak/test/framework/injection/Registry.java

```diff
@@ -176,11 +185,12 @@ private void matchDeployedInstancesWithRequestedInstances() {
     }
 
     private void deployRequestedInstances() {
+        requestedInstances.sort(RequestedInstanceComparator.INSTANCE);
```

Comment:
parallel execution readiness?

Reply:
Not sure what you mean, but for parallel execution we'll need to rewrite quite a bit of stuff in the registry for that to work.